api/src/pcapi/core/offerers/api.py

```diff
@@ -1489,26 +1489,55 @@ def list_users_offerers_to_be_validated_legacy(filter_: list[dict[str, typing.An
 
 
 def list_users_offerers_to_be_validated(
+    q: str | None,  # search query
     tags: list[offerers_models.OffererTag] | None = None,
     status: list[ValidationStatus] | None = None,
     offerer_status: list[ValidationStatus] | None = None,
     from_datetime: datetime | None = None,
     to_datetime: datetime | None = None,
 ) -> sa.orm.Query:
-    query = offerers_models.UserOfferer.query.options(
-        sa.orm.joinedload(offerers_models.UserOfferer.user),
-        sa.orm.joinedload(offerers_models.UserOfferer.offerer)
-        .joinedload(offerers_models.Offerer.action_history)
-        .joinedload(history_models.ActionHistory.authorUser),
-        sa.orm.joinedload(offerers_models.UserOfferer.offerer)
-        .joinedload(offerers_models.Offerer.UserOfferers)
-        .joinedload(offerers_models.UserOfferer.user),
+    query = (
+        offerers_models.UserOfferer.query.options(
+            sa.orm.joinedload(offerers_models.UserOfferer.user),
+            sa.orm.joinedload(offerers_models.UserOfferer.offerer)
+            .joinedload(offerers_models.Offerer.action_history)
+            .joinedload(history_models.ActionHistory.authorUser),
+            sa.orm.joinedload(offerers_models.UserOfferer.offerer).joinedload(offerers_models.Offerer.UserOfferers),
+            sa.orm.joinedload(offerers_models.UserOfferer.offerer)
+            .joinedload(offerers_models.Offerer.UserOfferers)
+            .joinedload(offerers_models.UserOfferer.user),
```

Comment:
(performance) Certains appels à joinedload sont redondants et génèrent des jointures identiques qui ne sont pas nécessaires : 

```suggestion
            sa.orm.joinedload(offerers_models.UserOfferer.offerer)
            .joinedload(offerers_models.Offerer.UserOfferers)
            .joinedload(offerers_models.UserOfferer.user),
```

De plus, je me demande pourquoi on fait cela : pour chaque UserOfferer, on charge la structure liée (ce qui me semble raisonnable), mais aussi les autres UserOfferers liées à cette structure (et leur User correspondant). A-t-on vraiment besoin de ces informations-là ?